test/scales/timeScaleTests.ts

```diff
@@ -1,178 +1,256 @@
+      });
+
+      it("produces correct number of ticks", () => {
+        // 100 year span
+        scale.domain([new Date(2000, 0, 1, 0, 0, 0, 0), new Date(2100, 0, 1, 0, 0, 0, 0)]);
+        let yearTicks = scale.tickInterval(Plottable.TimeInterval.year);
+        assert.strictEqual(yearTicks.length, 101, "generated correct number of ticks for 100 year span, every year");
+        // 1 year span
+        scale.domain([new Date(2000, 0, 1, 0, 0, 0, 0), new Date(2000, 11, 31, 0, 0, 0, 0)]);
+        let monthTicks = scale.tickInterval(Plottable.TimeInterval.month);
+        assert.strictEqual(monthTicks.length, 12, "generated correct number of ticks for 1 year span, every month");
+        let threeMonthTicks = scale.tickInterval(Plottable.TimeInterval.month, 3);
+        assert.strictEqual(threeMonthTicks.length, 4, "generated correct number of ticks for 1 year span, every 3 months");
+        // 1 month span
+        scale.domain([new Date(2000, 0, 1, 0, 0, 0, 0), new Date(2000, 1, 1, 0, 0, 0, 0)]);
+        let dayTicks = scale.tickInterval(Plottable.TimeInterval.day);
+        assert.strictEqual(dayTicks.length, 32, "generated correct number of ticks for 1 month span, every day");
+        // 1 day span
+        scale.domain([new Date(2000, 0, 1, 0, 0, 0, 0), new Date(2000, 0, 1, 23, 0, 0, 0)]);
+        let hourTicks = scale.tickInterval(Plottable.TimeInterval.hour);
+        assert.strictEqual(hourTicks.length, 24, "generated correct number of ticks for 1 day span, every hour");
+        // 1 hour span
+        scale.domain([new Date(2000, 0, 1, 0, 0, 0, 0), new Date(2000, 0, 1, 1, 0, 0, 0)]);
+        let minuteTicks = scale.tickInterval(Plottable.TimeInterval.minute);
+        assert.strictEqual(minuteTicks.length, 61, "generated correct number of ticks for 1 hour span, every minute");
+        let tenMinuteTicks = scale.tickInterval(Plottable.TimeInterval.minute, 10);
+        assert.strictEqual(tenMinuteTicks.length, 7, "generated correct number of ticks for 1 hour span, every 10 minutes");
+        // 1 minute span
+        scale.domain([new Date(2000, 0, 1, 0, 0, 0, 0), new Date(2000, 0, 1, 0, 1, 0, 0)]);
+        let secondTicks = scale.tickInterval(Plottable.TimeInterval.second);
+        assert.strictEqual(secondTicks.length, 61, "generated correct number of ticks for 1 minute span, every second");
+      });
     });
 
-  it("can be padded", () => {
-    let scale = new Plottable.Scales.Time();
-    scale.padProportion(0);
-    let unpaddedDomain = scale.domain();
-    scale.addIncludedValuesProvider((scale: Plottable.Scales.Time) => unpaddedDomain);
-    scale.padProportion(0.1);
-    assert.operator(scale.domain()[0].getTime(), "<", unpaddedDomain[0].getTime(), "left side of domain was padded");
-    assert.operator(scale.domain()[1].getTime(), ">", unpaddedDomain[1].getTime(), "right side of domain was padded");
-  });
+    describe("Auto Domaining", () => {
```

Comment:
everything else looks good except this describe block description 
there's tests for autoDomain(), domainMin(), and domainMax() here